docs/pages/concepts/logical/environments.md

```diff
@@ -8,12 +8,25 @@ language: en
 list: include
 ---
 
+Environments, in the context of Meshery, serve as logical groupings of connections and credentials. Connections are the crucial aspect of your cloud-native applications, ranging from Kubernetes clusters to Prometheus instances, Jaeger services, Nginx deployments, and more. These connections could be either managed or unmanaged, depending on the tiers of discovery performed by [MeshSync](concepts/architecture/meshsync), Meshery's dynamic, and sophisticated resource discovery component. The ability to organize these resources into logical groups brings a new level of convenience and efficiency to managing your cloud-native infrastructure.
```

Comment:
```suggestion
Meshery Environments allow you to logically group related [Connections]({{site.baseurl}}/concepts/logical/connections) and their associated [Credentials]({{site.baseurl}}/concepts/logical/credentials). Environments make it easier for you to manage, share, and work with a collection of resources as a group, instead of dealing with all your Connections and Credentials on an individual basis. 

### Assigning Resources 

Assign any number of Connections to an environment whether that Connection is managed or unmanaged (see [MeshSync](concepts/architecture/meshsync) to learn more about managed and unmanaged Connections). In-turn, assign any number of Environments to one or more [Workspaces](/concepts/logical/workspaces). Connections (and any associated Credentials) that are assigned to an Environment become immediately available for use in any associated Workspace.

### Sharing Resources between Environments

Environments can share resources. For example, you might create an environment named "production" and assign three connections: a GitHub connection, a Kubernetes connection, and a Prometheus connection. Subsequently, you also define a an environment named "dev/test "and assign three connections: a different Cooper, Netties connection, a different Prometheus connection, _and_ the same GitHub connection that is also assigned to the "production" environment.


## Summary

Environments represent a collection of resources in the form of Connections - both of managed and unmanaged Connections. Environment resources are comprised of Connections (and implicitly any Credentials used by those assigned Connections). Create and use environments to organize your connections and credentials into groups, and then make these resources available to you and your teams by assigning environments to Workspaces.
```